src/styles/globals.css

```
@@ -128,13 +133,15 @@
         rgba(254, 255, 230, 0.38) 79.52%
       );
   }
+  /* 홈 버블*/
```

Stylelint (17.2.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)